Assets/Scripts/Framework/Gameplay/Interactions/Grabbables/Grabbable.cs

```diff
@@ -1,12 +1,15 @@
 using FishNet.Object.Synchronizing;
+using UnityEngine;
 
 using DerailedDeliveries.Framework.Gameplay.Player;
+using FishNet.Object;
 
 namespace DerailedDeliveries.Framework.Gameplay.Interactions.Grabbables
 {
     /// <summary>
     /// A Interactable class that is used for all grabbable Interactables.
     /// </summary>
+    [RequireComponent(typeof(BoxCollider))]
```

Comment:
Maybe it is better to move this `RequireComponent` to the base class `Interactable` since the `Interactable` also needs a collider to work properly.